tabbycat/breakqual/aida.py

```diff
@@ -130,8 +130,16 @@ def reinsert_capped_teams(self):
 
 
 @register
-class Aida2016AustralsBreakGenerator(BaseAida2016BreakGenerator):
-    key = "aida-2016-australs"
+class Aida2016Australs3CapBreakGenerator(BaseAida2016BreakGenerator):
+    key = "aida-2016-australs-3cap"
 
     def reinsert_capped_teams(self):
         self._reinsert_capped_teams(self.capped_teams)
+
+@register
+class Aida2016Australs4CapBreakGenerator(BaseAida2016BreakGenerator):
+    key = "aida-2016-australs-4cap"
+    institution_cap = 4
+    
+    def reinsert_capped_teams(self):
+        self._reinsert_capped_teams(self.capped_teams)
```

Comment:
Since `reinsert_capped_teams` is exactly the same, it's probably better practice to create a new class in the hierarchy, say, `BaseAida2016AustralsBreakGenerator`, from which `Aida2016Australs3CapBreakGenerator` and `Aida2016Australs4CapBreakGenerator` both derive. 🙂